libdebug/memory/chunked_memory_view.py

```diff
@@ -112,3 +113,47 @@ def write(self: ChunkedMemoryView, address: int, data: bytes) -> None:
                 address + size - remainder,
                 data[size - remainder :] + prev_data[remainder:],
             )
+
+    @property
+    def maps(self: ChunkedMemoryView) -> MemoryMapList:
+        """Returns a list of memory maps in the target process.
+
+        Returns:
+            MemoryMapList: The memory maps.
+        """
+        return self._internal_debugger.maps
+
+    def resolve_address(
+        self: AbstractMemoryView,
+        address: int,
+        backing_file: str,
+        skip_absolute_address_validation: bool = False,
+    ) -> int:
+        """Normalizes and validates the specified address.
+
+        Args:
+            address (int): The address to normalize and validate.
+            backing_file (str): The backing file to resolve the address in.
+            skip_absolute_address_validation (bool, optional): Whether to skip bounds checking for absolute addresses. Defaults to False.
+
+        Returns:
+            int: The normalized and validated address.
+
+        Raises:
+            ValueError: If the substring `backing_file` is present in multiple backing files.
+        """
+        return self._internal_debugger.resolve_address(
+            address, backing_file, skip_absolute_address_validation,
+        )
+
+    def resolve_symbol(self: AbstractMemoryView, symbol: str, backing_file: str) -> int:
```

Comment:
`resolve_symbol` for this file and for direct_memory_view seems to do the same thing, why the need for two different concrete implementations? It should stay in the abstract_memory_view for now.